packages/node_modules/@ciscospark/redux-module-media/src/reducer.js

```diff
@@ -52,7 +45,16 @@ export default function reducer(state = initialState, action) {
   case UPDATE_STATUS:
     return state.mergeIn([`status`], action.payload.status);
   case UPDATE_CALL_STATE: {
+    let remoteAudioMuted, remoteVideoMuted;
     const call = action.payload.call;
+    try {
+      remoteAudioMuted = call.remoteAudioMuted;
```

Comment:
Yup, turns out `change` events get fired on the `call` object before `call.remoteAudioMuted` and `call.remoteVideoMuted` are available as derived values, so we get an Uncaught Type Error. There should be a fix for the SDK, but for now this is a workaround.